src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1649,38 +1672,30 @@ private bool IsWindows()
             return Path.DirectorySeparatorChar == '\\';
```

Comment:
The `IsWindows()` method should be deleted - it was part of workaround that you have removed.